introduction.md

```diff
@@ -1,2 +1,6 @@
 This is an introduction to the dojo.
-My favorite codeschool path is Ruby.
\ No newline at end of file
+My favorite codeschool path is Ruby.
+
+Deadly Skills
+*Git
+*Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?